docs/common/vars.rst

```diff
@@ -4,6 +4,7 @@
 .. |doca-driver-version| replace:: doca3.2.0-25.10-1.1.2.0-0
 .. |doca-driver-repository| replace:: nvcr.io/nvstaging/mellanox
 .. |doca-driver-version-lts| replace:: 24.10-0.7.0.0-0
+.. |doca-driver-version-lts| replace:: 24.10-0.7.0.0-0
```

Comment:
Once the template is fixed, need to run `make generate-docs-versions-var` to generate this file from the template